### PR TITLE
Remove undocumented, old, and unused debug commands

### PR DESCRIPTION
**Pull request by DL6ER**
**By submitting this pull request, I confirm the following (please check boxes, eg [X]) _Failure to fill the template will close your PR_:**

***Please submit all pull requests against the `development` branch. Failure to do so will delay or deny your request***

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] I have checked that [another pull request](https://github.com/pi-hole/FTL/pulls) for this purpose does not exist.
- [X] I have considered, and confirmed that this submission will be valuable to others.
- [X] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [X] I give this submission freely, and claim no ownership to its content.

**How familiar are you with the codebase?:** 

## 10

---

Remove undocumented, old, and unused debug commands. (unused = I haven't used them for > 1 year, they have been helpful in the very early days of FTL < v1.0).

Also remove reading of `API_PRIVACY_MODE` as this was already ignored before. This option is superseded by the new privacy levels.

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

### CODE REVIEW

File: database.c [hunk 1]
```diff
@@ -63,15 +63,15 @@ double get_db_filesize(void)
...
+/*
 static int callback(void *NotUsed, int argc, char **argv, char **azColName){
 	int i;
 	for(i=0; i<argc; i++){
 		logg("%s = %s", azColName[i], argv[i] ? argv[i] : "NULL");
 	}
 	return 0;
 }
-
+*/
```
Comment (Mcat12) [inline database.c]: Git will remember this code, so it can be removed instead of commented out.

File: database.c [hunk 2]
```diff
@@ -102,10 +102,9 @@ bool dbquery(const char *format, ...)
 		return false;
 	}
 
-	if(debugDB)
-		rc = sqlite3_exec(db, query, callback, NULL, &zErrMsg);
-	else
-		rc = sqlite3_exec(db, query, NULL, NULL, &zErrMsg);
+//	Use callback when debugging
+//	rc = sqlite3_exec(db, query, callback, NULL, &zErrMsg);
```
Comment (Mcat12) [inline database.c]: See above.

File: socket.c [hunk 3]
```diff
@@ -300,7 +300,6 @@ void *telnet_connection_handler_thread(void *socket_desc)
 	// Set connection type to telnet
 	istelnet[sock] = true;
 
-	// Store copy only for displaying the debug messages
 	int sockID = sock;
```
Comment (Mcat12) [inline socket.c]: You can replace the only remaining use of `sockID` with `sock`.

**Comment by DL6ER** [comment]
Merge conflicts resolved

**Pull request merged**